extra-utils/ccat/autobuild/build

```diff
@@ -0,0 +1,14 @@
+declare -A ARCH_MAP;
+ARCH_DEF=$(dpkg --print-architecture)
```

Comment:
Please, do not use dpkg for architecture detection, use the Autobuild defined `${CROSS:-$ARCH}`.

Reply:
ok